pkg/controller/usersignup/masteruserrecord.go

```diff
@@ -37,7 +37,8 @@ func migrateOrFixMurIfNecessary(mur *toolchainv1alpha1.MasterUserRecord, nstempl
 			changed = true
 		}
 	}
-	// migration for CRT-1075: add an annotation with the email address (same as for associated UserSignup resource)
+	// Migration for CRT-1075: add an annotation with the email address (same as for associated UserSignup resource)
+	// TODO: remove as part of CRT-1074
```

Comment:
no, indeed. I remove the comment on line 40 in 8e785d8

Reply:
ok, I see what you mean: the `if mur.Annotations == nil || mur.Annotations[toolchainv1alpha1.MasterUserRecordEmailAnnotationKey] == ""` statement should return `false` now, so this code block is not used anymore, but as the comment above says, I intend to clean up all the migration code in https://issues.redhat.com/browse/CRT-1074, which will be the last subtask of this story. 

I though it would be better to keep the PR focused on a single subject, but let me know if you prefer that I address this clean-up immediatly.